protos/src/main/protobuf/blockstream.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+option java_package = "com.hedera.block.protos";
+option java_outer_classname = "BlockStreamServiceGrpcProto";
+
+service BlockStreamGrpc {
+    rpc StreamSink(stream Block) returns (stream BlockResponse) {}
+    rpc StreamSource(stream BlockResponse) returns (stream Block) {}
+}
+
+message Block {
+    int64 id = 1;
+    string value = 2;
+}
+
+message BlockResponse {
+    int64 id = 1;
+}
+
+message Empty {
+}
```

Comment:
Since we're starting fresh, let's start right off with writing the API specification here.

Reply:
Can you point me to an example of a gRPC API spec to emulate?

Reply:
https://github.com/hashgraph/hedera-protobufs/blob/main/services/consensus_create_topic.proto

Reply:
I added API specification documentation for this proto file.